ui/components/AutomationDetail.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from "react";
 import { useRouteMatch } from "react-router-dom";
 import styled from "styled-components";
-import { Automation, useSyncFluxObject } from "../hooks/automations";
+import { useSyncFluxObject } from "../hooks/automations";
 import { useToggleSuspend } from "../hooks/flux";
 import { useGetObject } from "../hooks/objects";
 import { FluxObjectKind } from "../lib/api/core/types.pb";
-import { fluxObjectKindToKind } from "../lib/objects";
 import { getSourceRefForAutomation } from "../lib/utils";
```

Comment:
Unused import. If this function is not needed anymore since you added sourceRef getter to the classes, is it possible to remove it completely from utils and tests and replace it with the sourceRef getter in `AutomationsTable` too?

Reply:
I see other unused imports, ah, it's because the tests have not completed yet because of the merging/rebase conflict.

Reply:
> is it possible to remove it completely from utils and tests and replace it with the sourceRef getter in AutomationsTable and SourceDetail too?

I left rewriting `AutomationsTable` out of this PR, so not yet.

This will also apply to `FluxObjectKind` and everything that uses that, which ripples through the code base, so I cut scope.